dist/index.html

```diff
@@ -1,17 +1,42 @@
 <!DOCTYPE html>
 
 <html lang="en">
+  
   <head>
-    <meta charset="utf-8" />
-
-    <title>SHOT RUN</title>
-    <meta name="description" content="SHOT RUN" />
+    <meta charset="UTF-8" />
+    <!-- Update title to "Sober Run" not "Shot Run"-->
```

Comment:
lets try to stray away from too many comments. if the code looks self-explanatory/clear enough, it doesn't need a comment.

i would usually only comment if its:
- a temporary change
- complicated logic
- a weird "gotcha", or bug that you don't want other developers to run into